lib/y2logsstep.pm

```diff
@@ -70,7 +70,7 @@ sub record_dependency_issues {
 }
 
 sub save_upload_y2logs() {
-    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2";
+    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2", 540;
```

Comment:
Please, no, we upload yast logs all the time, I really do not like the idea of a 5 and a half minute timeout on such a regularly used function.